MMM-Spotify.js

```diff
@@ -294,7 +368,13 @@ Module.register("MMM-Spotify", {
     sDom.classList.remove("noPlayback")
 
     const cover_img = document.getElementById("SPOTIFY_COVER_IMAGE")
-    const img_url = playbackItem.album.images[0].url
+    let img_index = 0
+    // cover data is stored in 3 sizes. let's fetch the appropriate size to reduce 
+    // bandwidth usage bsed on player style
+    if (this.config.style !== "default") {
+      img_index = this.enbaleMiniBar ? 2 : 1
```

Comment:
typo

= this.enableMiniBar 

also, I checked on larger screens 2 is actually super blurry. Can we just default to 1?

Reply:
it's result of this:

https://github.com/bugsounet/MMM-Spotify/blob/dev/MMM-Spotify.js#L738-L745

so miniBar works ONLY with module position